eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -131,8 +137,10 @@ def fitJWST(eventlabel, ecf_path='./', s4_meta=None):
                 flux = np.ma.masked_array([])
                 flux_err = np.ma.masked_array([])
                 for channel in range(chanrng):
-                    flux = np.ma.append(flux,meta.lcdata[channel,:] / np.mean(meta.lcdata[channel,:]))
-                    flux_err = np.ma.append(flux_err,meta.lcerr[channel,:] / np.mean(meta.lcdata[channel,:]))
+                    flux = np.ma.append(flux,lc.data.values[channel,:] / \
+                                np.mean(lc.data.values[channel,:]))
```

Comment:
I'm pretty sure the code in main uses np.ma.mean here to avoid issues with masked data. The issue though is that lc.data isn't a masked array, so the np.ma calls here aren't likely to do much

Reply:
For now can you use np.nanmean to at least avoid any issues caused by NaNs